schema.ts

```diff
@@ -1245,6 +1246,18 @@ export interface ObjectSchemaDefinition<K extends string, L extends string>
    * @hidden
    */
   groupIdProperty?: PropertyIdentifier<K>;
+
+  /**
+   * The name of the property within {@link ObjectSchemaDefinition.properties} that can be be interpreted as
+   * a large blob of unstructured content.
+   *
+   * This should consist of text or markdown content.
+   *
+   * Must be a {@link ValueType.String} property.
+   * TODO(sam): Unhide this
+   * @hidden
+   */
+  unstructuredContentProperty?: PropertyIdentifier<K>;
```

Comment:
@patrick-codaio a big question here is generally this content will NOT work in a sync table (will always blow up the cell size limit). Is that an issue?

Reply:
I suspect it will work ~2/3 of the time, so I vote for not doing anything about this now (other than maybe a build error that a pack maker should not make a field like this a featuredProperty?). We can write code for docs to ignore these columns if we decide to later.

Reply:
oh, and also think of a future of snowflake-backed sync tables in docs, this data would be more useful then.